### PR TITLE
Support reading of actual warning in SAVE from the file

### PR DESCRIPTION
**Pull request by Cheshiriks**
What's done:
* Added reading of actual warning from the file

### CODE REVIEW

File: save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPlugin.kt [hunk 1]
```diff
@@ -146,7 +147,12 @@ class WarnPlugin(
             return failTestResult(originalPaths, ex, execCmd)
         }
 
-        val actualWarningsMap = collectActualWarningsWithLineNumbers(result, warnPluginConfig)
+        val actualWarningsMap = if (warnPluginConfig.actualWarningsFileName != null) {
```
Comment (akuleshov7) [inline save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPlugin.kt:150]: may be `let` here (as suggested in diktat)?

**Review by akuleshov7 — APPROVED** [review]

File: save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPlugin.kt (hunk 1 above)
Comment (Cheshiriks) [reply save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPlugin.kt]: Yes, fixed

**Comment by Cheshiriks** [comment]
> What about test?

It is there, but I did not add it because of the full path to the file

**Comment by kgevorkyan** [comment]
If `okio` still not support `absolute`, let's at least create draft PR with your test, after merging of this one

**Review by kgevorkyan — APPROVED** [review]

**Comment by Cheshiriks** [comment]
> If `okio` still not support `absolute`, let's at least create draft PR with your test, after merging of this one

Ok

**Pull request merged**